app/assets/javascripts/discourse/app/lib/bulk-select-helper.js

```diff
@@ -35,6 +35,10 @@ export default class BulkSelectHelper {
     this.clear();
   }
 
+  experimentalTopicBulkActionsEnabled() {
+    this.siteSettings.experimental_topic_bulk_actions;
```

Comment:
When the user serializer change is made, the `currentUser` will need to be used here